internal/exec/util/file.go

```diff
@@ -225,3 +226,24 @@ func (u Util) GetUserGroupID(l *log.Logger, user types.NodeUser, group types.Nod
 func MkdirForFile(path string) error {
 	return os.MkdirAll(filepath.Dir(path), DefaultDirectoryPermissions)
 }
+
+func (u Util) DeletePathOnOverwrite(n types.Node) error {
+	if !n.Overwrite {
+		return nil
+	}
+	path := u.JoinPath(string(n.Path))
+	_, err := os.Stat(path)
+	switch {
+	case os.IsNotExist(err):
+		break
+	case err != nil:
+		return fmt.Errorf("error stating %q: %v", n.Path, err)
```

Comment:
nit: `error stat()ing` or `error statting` or change it to something that isn't a different english word